raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -170,6 +170,14 @@ def matrix(self, data):
                 composite_filters=composite_filters,
                 include_original_columns_only=False)
 
+            msg = "Feature {} not found in dataset. Existing features: {}"
+            for feature in features:
+                if feature is None:
+                    continue
+                if feature not in filtered_data_df.columns:
+                    raise ValueError(
```

Comment:
Can we raise UserConfigValidationException instead of ValueError here. This should be available in the new raiutils package.

Reply:
done!